discussion/src/main/java/org/openedx/discussion/presentation/comments/DiscussionCommentsViewModel.kt

```diff
@@ -300,11 +300,12 @@ class DiscussionCommentsViewModel(
                 )
                 val response = interactor.createComment(thread.id, rawBody, null, reCaptchaToken)
                 response.isAuthor = response.author == corePreferences.user?.username
-
+                response.shouldHighlight = commentId.isEmpty()
```

Comment:
A few things that are wrong with this statement:

- Why should we highlight the response if the comment ID is empty? Both variables are independent of each other. 😄 
- Though it does the trick, as per the ticket description, we don't need to highlight a response; we only need to scroll up to it. 

You need to create a new stateFlow and mimic the functionality without highlighting the response.